addons/website_sale/views/templates.xml

```diff
@@ -1212,5 +1212,207 @@
+                    <strong>Date:</strong> <span t-field="o.create_date" t-field-options='{"widget": "date"}'/>
+                </div>
+                <div class='row'>
+                  <div class="col-md-6">
+                    <div>
+                      <strong>Invoicing Address</strong>
+                    </div>
+                    <div>
+                      <address t-field="o.partner_invoice_id" t-field-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}'/>
+                    </div>
+                    <t t-set="invoices" t-value="[i for i in o.invoice_ids if i.state not in ['draft', 'cancel']]"/>
+                    <t t-if="invoices">
+                      <div>
+                        <strong>Invoices</strong>
+                      </div>
+                      <div>
+                        <t t-foreach="invoices" t-as="i">
+                          <t t-set="report_url" t-value="'/report/pdf/account.report_invoice/%s' % i.id"/>
+                          <div>
+                            <a t-att-href="report_url"><span class="fa fa-download"/></a>
+                            <a t-att-href="report_url"><span t-field="i.number"/></a>
+                            <span class="text-muted" t-field="i.date_invoice"/>
+                            <t t-if="i.state == 'paid'">
+                              <span class="label label-success label-text-align"><i class="fa fa-fw fa-check"/> Paid</span>
+                            </t>
+                            <t t-if="i.state != 'paid'">
+                              <span class="label label-info label-text-align"><i class="fa fa-fw fa-clock-o"/> Waiting</span>
+                            </t>
+                          </div>
+                        </t>
+                      </div>
+                    </t>
+                  </div>
+                  <div id="shipping_address" class="col-md-6">
+                    <div>
+                      <strong>Shipping Address</strong>
+                    </div>
+                    <div>
+                      <address t-field="o.partner_shipping_id" t-field-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}'/>
+                    </div>
+                  </div>
+                </div>
+
+                <hr/>
+
+                <div id="product_lines">
+                  <!-- Product lines come here -->
+                </div>
+
+                <hr/>
+
+                <div class="row">
+                  <div class="col-md-6">
+                    <div>
+                      <strong>Contact</strong>
+                    </div>
+                    <div t-field="o.user_id.partner_id" t-field-options='{"widget": "contact", "fields": ["email", "phone"]}'/>
+                  </div>
+                  <div class="col-md-6">
+                    <div class="row">
+                      <div class="col-md-10 text-right">
+                        Untaxed Amount :
```

Comment:
Just to split hairs in four: in English, you do not have to put a space before a colon, that's French typo ;-)